www-client/uget/uget-2.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
+
+IUSE="+gnutls aria2 gstreamer appindicator libnotify nls openssl sockets rss"
+KEYWORDS="~amd64 ~arm ~x86"
+SRC_URI="mirror://sourceforge/urlget/${P}.tar.gz"
+
+DESCRIPTION="Download manager using gtk+ and libcurl"
+HOMEPAGE="http://www.ugetdm.com"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+
+DOCS=( AUTHORS ChangeLog NEWS README )
```

Comment:
Is this different than the result of installing via default list?

Reply:
NEWS is not installed by default.

Reply:
Are you sure about that? It's there in the code.